packages/inbound-email-handler/src/index.ts

```diff
@@ -68,6 +68,19 @@ const saveReceivedEmail = async (
   return response.data as SaveReceivedEmailResponse
 }
 
+export const parsedTo = (parsed: Record<string, string>): string => {
+  // envelope to contains the real recipient email address
+  try {
+    const envelope = JSON.parse(parsed.envelope) as Record<
+      string,
+      string | string[]
+    >
+    return envelope.to.toString().split(',')[0] || parsed.to
```

Comment:
What is the split for here, looking at the sample envelope I saw it was like this `{"to":["jacksonh-XXXXXXXX@inbox.omnivore.app"]`

Reply:
yeah, could be removed. i was considering the case where `to` is a string instead of an array

Reply:
Oh, I see, so there can be multiple? Do you know if sendgrid always handles that?

Reply:
<img width="763" alt="Screenshot 2023-03-23 at 6 31 01 PM" src="https://user-images.githubusercontent.com/12963965/227176406-c11c13d0-6527-4019-9e11-9892a0b09472.png">

Reply:
yeah

Reply:
I updated the code to use the first element in the array